segway/input_master.py

```diff
@@ -0,0 +1,503 @@
+#!/usr/bin/env python
+
```

Comment:
For the next round, can you please fix this so that Git knows what it should diff against? It looks like this is modification of old code and I don't think anyone wants us to review it all de novo

Reply:
This file was previously `segway_input_master.py`. The helper functions and opening section are the only things which were not changed. I haven't yet found a way to change how Git generates the diff, but will at minimum specify the relevant line sections in the next round.